.github/workflows/test.yml

```diff
@@ -66,6 +66,19 @@ jobs:
         name: sample_wheels
         path: wheelhouse
 
+    - name: Get some sample wheels (GitHub Action)
+      uses: ./
+      with:
+        package-dir: sample_proj
+        output-dir: wheelhouse_action
+      env:
+        CIBW_ARCHS_MACOS: x86_64 universal2 arm64
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: sample_wheels_action
+        path: wheelhouse_action
+
```

Comment:
There was no test for the GitHub Action so I duplicated the sample wheels steps to test the action.

Reply:
It might be worth isolating those tests a bit more (and maybe reduce them a bit as well in this case to avoid duplicate docker pulling / python installation).

Reply:
Its probably worth consolidating these Sample Build steps. I'm happy to use the Github Action one as the version we keep, given that the tests are checking the command-line invocation.

Reply:
I've pushed a change to that effect.